package.json

```diff
@@ -62,33 +62,21 @@
     "wdio-visual-regression-service": "^0.8.0"
   },
   "devDependencies": {
-    "autoprefixer": "^6.7.7",
     "babel-cli": "^6.24.1",
     "babel-core": "^6.24.1",
-    "babel-loader": "^7.0.0",
     "babel-plugin-transform-object-assign": "^6.22.0",
     "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-polyfill": "^6.23.0",
     "babel-preset-es2015": "^6.24.1",
-    "css-loader": "^0.28.7",
     "danger": "^1.0.0",
     "eslint": "^3.19.0",
     "eslint-config-airbnb": "^14.1.0",
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^4.0.0",
     "eslint-plugin-react": "^6.9.0",
-    "extract-text-webpack-plugin": "^2.1.0",
-    "file-loader": "^0.11.1",
     "html-webpack-plugin": "^2.30.0",
-    "json-loader": "^0.5.4",
     "nightwatch": "^0.9.12",
-    "node-sass": "^4.5.2",
-    "postcss-loader": "^2.0.6",
-    "raw-loader": "^0.5.1",
     "rimraf": "^2.6.1",
-    "sass-loader": "^6.0.6",
     "shelljs": "^0.7.6",
-    "style-loader": "^0.19.0",
     "webdriverio": "^4.9.8",
     "webpack": "^3.6.0",
     "webpack-dev-server": "2.7.1",
```

Comment:
Looks like this can be removed too since we're removing `devServer` config from `test.config`

Reply:
It's still referenced here in the webpack dev server service and people using toolkit 2.0 and wdio will still be using this because the config to switch to the express dev server service is in their packages :( . https://github.com/cerner/terra-toolkit/blob/master/src/wdio/services/WebpackDevServerService.js